client/packages/common/src/intl/locales/en/dispensary.json

```diff
@@ -59,5 +59,6 @@
   "messages.error-saving-prescription": "Error saving prescription 🥺",
   "placeholder.search-by-first-name": "Search by first name",
   "placeholder.search-by-last-name": "Search by last name",
-  "placeholder.search-by-identifier": "Search by id"
+  "placeholder.search-by-identifier": "Search by id",
```

Comment:
the hook `usePrescriptionDeleteLines` is using this translation file and calls 
```
      deleteSuccess: t('messages.deleted-lines', {
```

but there is not deleted-lines message in dispensary.json. Might have to move to common.json too